tests/foreman/cli/test_activationkey.py

```diff
@@ -1581,3 +1581,65 @@ def test_positive_invalid_release_version(module_sca_manifest_org, module_target
         }
     )
     assert update_ak[0]['message'] == 'Activation key updated.'
+
+
+# -------------------------- MULTI-CV SCENARIOS -------------------------
+def test_positive_multi_cv_info(
+    session_multicv_sat, session_multicv_org, session_multicv_default_ak
+):
+    """Verify that multi-env AK display into hammer activation-key info commands output
+
+    :id: 6a1c3189-74f9-4a54-8579-f3b045870cd9
+
+    :steps:
+        1. Create two lifecycle environments and two content views, publish/promote to respective lce
+        2. Create activation key and update ak with multiple content view environments
+        3. Check ak info displays 'Multi Content View Environment' and 'Content View Environments'
+
+    :expectedresults: AK info displays 'Multi Content View Environment' and 'Content View Environments'
+
+    :CaseImportance: Medium
+
+    :CaseComponent: ActivationKeys
+
+    :team: Phoenix-subscriptions
+
+    :parametrized: No
+
+    :Verifies: SAT-27863
+    """
+    # Create two lifecycle environments
+    lce1 = session_multicv_sat.api.LifecycleEnvironment(organization=session_multicv_org).create()
+    lce2 = session_multicv_sat.api.LifecycleEnvironment(organization=session_multicv_org).create()
+
+    # Create two content views
+    cv1 = session_multicv_sat.api.ContentView(organization=session_multicv_org).create()
+    cv1.publish()
+    cv1 = cv1.read()
+    cv1.version[0].promote(data={'environment_ids': lce1.id})
+
+    cv2 = session_multicv_sat.api.ContentView(organization=session_multicv_org).create()
+    cv2.publish()
+    cv2 = cv2.read()
+    cv2.version[0].promote(data={'environment_ids': lce2.id})
```

Comment:
```suggestion
    # Creation of 2 LCEs
    lces_list = [ session_multicv_sat.api.LifecycleEnvironment(organization=session_multicv_org).create() for i in 
    range(2)
    ]
    # Creation of 2 CVs
    cvs_list = [session_multicv_sat.api.ContentView(organization=session_multicv_org).create() for i in range(2)
    ]

    # Publish, read and promote both CVs
    for i in range(2):
        cvs_list[i].publish()
        cvs_list[i] = cvs_list[i].read()
        cvs_list[i].version[0].promote(data={'environment_ids': lces_list[i].id})
```

Reply:
I appriciate your suggested changes those could be implemented, only need to test locally.
From feature understanding point of view, keeping code separated would be good at this moment as we are in the initial stage of implementing multi-cv stuff.